yast_rdp.pm

```diff
@@ -0,0 +1,90 @@
+# SUSE's openQA tests
```

Comment:
this file is for ??

Reply:
you know, for yast2-rdp, there are two modes, one with gui, and the other is cmdline mode, like this "yast2 rdp allow set=yes" to start xrdp service. yast_rdp.pm is for testing in cmdline mode.